code/modules/mob/living/basic/trader/trader_data.dm

```diff
@@ -0,0 +1,138 @@
+///Used to contain the traders initial wares, and speech
+/datum/trader_data
+
+	///The item that marks the shopkeeper will sit on
+	var/shop_spot_type =  /obj/structure/chair/plastic
+	///The sign that will greet the customers
+	var/sign_type = /obj/structure/trader_sign
+	///Sound used when item sold/bought
+	var/sell_sound = 'sound/effects/cashregister.ogg'
+	///The currency name
+	var/currency_name = "credits"
+	///The initial products that the trader offers
+	var/list/initial_products = list(
+		/obj/item/food/burger/ghost = list(PAYCHECK_CREW * 4, INFINITY),
+	)
+	///The initial products that the trader buys
+	var/list/initial_wanteds = list(
+		/obj/item/ectoplasm = list(PAYCHECK_CREW * 2, INFINITY, ""),
+	)
+	///The speech data of the trader
+	var/list/say_phrases = list(
+		ITEM_REJECTED_PHRASE = list(
+			"Sorry, I'm not a fan of anything you're showing me. Give me something better and we'll talk."
+		),
+		ITEM_SELLING_CANCELED_PHRASE = list(
+			"What a shame, tell me if you changed your mind."
+		),
+		ITEM_SELLING_ACCEPTED_PHRASE = list(
+		"Pleasure doing business with you."
```

Comment:
```suggestion
			"Pleasure doing business with you.",
```

let's be consistent

Reply:
Oh. I only fixed this in Mr Bones looks like it, I was confused how it came back.